openapi.yaml

```diff
@@ -143,374 +82,417 @@ paths:
       tags:
         - phones
       operationId: createPhones
       parameters:
-        - name: phoneBody
+        - name: body
           in: body
-          description: Post body for the new Phone
+          description: Post body for the new phone
           required: true
           schema:
-            $ref: '#/definitions/NewPhoneResultObject'
-        - $ref: '#/parameters/osuID'
-        - $ref: '#/parameters/authorization'
+            $ref: '#/definitions/PhonePostBody'
+        - $ref: '#/parameters/osuId'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/PhoneResult'
+        '400':
+          $ref: '#/responses/400'
+        '404':
+          $ref: '#/responses/404'
+        '500':
+          $ref: '#/responses/500'
+  /persons/{osuId}/emails:
+    get:
+      description: Get emails for a person
+      tags:
+        - emails
+      operationId: getEmails
+      parameters:
+        - $ref: '#/parameters/osuId'
+        - name: emailType
+          in: query
+          type: string
+          required: false
+          enum: &emailTypeEnums
+            - ONID
+            - STU
+            - VEND
+            - VWEB
+            - EMPL
+            - VPAY
+            - EMER
+          description: |
+            Type of email
+            - `STU` - Non-directory Student
+            - `ONID` - Student
+            - `VEND` - Vendor
+            - `VWEB` - Vendor
+            - `EMPL` - Employee
+            - `VPAY` - Vendor Pay
+            - `EMER` - Emergency Contact Persons Email for Student
       responses:
-        200:
+        '200':
           description: Successful response
           schema:
-            $ref: '#/definitions/PhoneResultObject'
-        400:
-          description: Bad request
+            $ref: '#/definitions/EmailSetResult'
+        '404':
+          $ref: '#/responses/404'
+        '500':
+          $ref: '#/responses/500'
+    post:
+      description: Create an email address for a person
+      tags:
+        - emails
+      operationId: createEmails
+      parameters:
+        - name: body
+          in: body
+          required: true
           schema:
-            $ref: '#/definitions/ErrorArray'
-        404:
-          description: Person not found
+            $ref: '#/definitions/EmailPostBody'
+        - $ref: '#/parameters/osuId'
+      responses:
+        '202':
+          description: Email successfully created
           schema:
-            $ref: '#/definitions/Error'
-
+            $ref: '#/definitions/EmailResult'
+        '400':
+          $ref: '#/responses/400'
+        '404':
+          $ref: '#/responses/404'
+        '500':
+          $ref: '#/responses/500'
 parameters:
-  osuID:
-    name: osuID
+  osuId:
+    name: osuId
     in: path
     required: true
-    description: OSU ID of person
+    description: OSU ID of a person
     type: string
```

Comment:
Can you add pattern here? https://github.com/osu-mist/students-api/blob/master/openapi.yaml#L366